Documentation/known-issues.md

```diff
@@ -0,0 +1,61 @@
+Known issues & workarounds
+==========================
```

Comment:
Can we make this more direct? Check out this page: https://support.google.com/a/answer/6166309?hl=en. It focuses on the known issues first, then talks to specifics about what they are.